steembase/http_client.py

```diff
@@ -184,90 +187,66 @@ def call(self,
 
         """
 
-        api = kwargs.get('api', None)
-        return_with_args = kwargs.get('return_with_args', None)
-        _ret_cnt = kwargs.get('_ret_cnt', 0)
-
-        body = HttpClient.json_rpc_body(name, *args, **kwargs)
-        response = None
+        return_with_args = kwargs.get('return_with_args', self.return_with_args)
```

Comment:
Does this need to be a pop? It is not removed from the kwargs and currently will be serialized into the body if it is set.

Reply:
Good catch, yep it does